### PR TITLE
[doc] PR for Update screenshots and procedures in create-universe-multi-zone-kubernetes.md

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-3611

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md [hunk 1]
```diff
@@ -42,100 +42,81 @@ Note that the cloud provider example used in this document has a cluster-level a
...
 Complete the **G-Flags** section as follows:
 
-- Use the **Master** fields to specify YugabyteDB Masters parameters, one parameter per field (see [YB Master Configuration Flags](/preview/reference/configuration/yb-master/#configuration-flags) for details).
+- Click **Add Flags > Add to Master** to specify YugabyteDB Masters parameters, one parameter per field.
 
-- Use the **T-Server** fields to specify the YugabyteDB T-Servers parameters, one parameter per field (see [YB T-Server Configuration Flags](/preview/reference/configuration/yb-tserver/#configuration-flags) for details).
+- Click **Add Flags > Add to T-Server** to specify the YugabyteDB T-Servers parameters, one parameter per field.
 
-  ![img](/images/yb-platform/kubernetes-config8.png)
+  For details, see the following:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:86]: Need a line break here? Runs in in Preview

**Review by ddhodge — COMMENTED** [review]
Minor comments. kubernetes-config01.png is included in PR but I don't see it in the doc.

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md [hunk 2]
```diff
@@ -42,100 +42,81 @@ Note that the cloud provider example used in this document has a cluster-level a
...
 
   ![img](/images/yb-platform/kubernetes-config5.png)
 
 Complete the **Instance Configuration** section as follows:
 
-- Use the **Instance Type** field to select the CPU and memory combination, as per needs to allocate the YB-TServer pods. The default is Small (4 Cores & 7.5GB RAM). You can override this setting when you configure the Kubernetes cloud provider (see [Configuring the region and zones](/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes/#configure-the-region-and-zones)).
-
-- In the **Volume Info** field, specify the number of volumes multiplied by size. The default is 1*100GB.
-
+- Use the **Instance Type** field to select the CPU and memory combination, as per needs to allocate the YB-TServer nodes. The default is small. You can override this setting when you configure the Kubernetes cloud provider (see [Configuring the region and zones](/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes/#configure-the-region-and-zones)).
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:64]: fixed

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md [hunk 3]
```diff
@@ -42,100 +42,81 @@ Note that the cloud provider example used in this document has a cluster-level a
...
+- In the **Volume Info** field, specify the number of volumes multiplied by size. The default is 1 x 100GB.
 - Use the **Enable YSQL** field to enable the YSQL API endpoint to run Postgres-compatible workloads. This setting is enabled by default.
-
 - Use the **Enable YEDIS** field to enable the YEDIS API endpoint to run REDIS-compatible workloads. This setting is disabled by default.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:67]: fixed

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:86]: fixed

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md [hunk 4]
```diff
@@ -42,100 +42,81 @@ Note that the cloud provider example used in this document has a cluster-level a
...
 
-- Use the **Instance Type** field to select the CPU and memory combination, as per needs to allocate the YB-TServer pods. The default is Small (4 Cores & 7.5GB RAM). You can override this setting when you configure the Kubernetes cloud provider (see [Configuring the region and zones](/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes/#configure-the-region-and-zones)).
-
-- In the **Volume Info** field, specify the number of volumes multiplied by size. The default is 1*100GB.
-
+- Use the **Instance Type** field to select the CPU and memory combination, as per needs to allocate the YB-TServer nodes. The default is small. You can override this setting when you configure the Kubernetes cloud provider (see [Configuring the region and zones](/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes/#configure-the-region-and-zones)).
+- In the **Volume Info** field, specify the number of volumes multiplied by size. The default is 1 x 100GB.
 - Use the **Enable YSQL** field to enable the YSQL API endpoint to run Postgres-compatible workloads. This setting is enabled by default.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md]: changed

**Pull request merged**